runtime/modules/java/core/src/test/java/io/atlasmap/java/inspect/ClassInspectionServiceTest.java

```diff
@@ -76,37 +75,6 @@ public void testDetectArrayClass() {
         assertEquals(Map.class, classInspectionService.detectArrayClass(Map[][][][].class));
     }
 
-    @Test
```

Comment:
I think these test cases should be covered from somewhere else. Same tests should be run via public methods.

Reply:
Can you please elaborate